component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mobileconnect/MobileConnectAuthenticator.java

```diff
@@ -84,10 +89,32 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
                                                  AuthenticationContext context)
             throws AuthenticationFailedException {
 
+//        Map<String, String> smsOTPParameters = getAuthenticatorConfig().getParameterMap();
```

Comment:
Remove commented code if not required. Otherwise adding a comment saying why you are commenting this out temporarily would be nice.